distribution/docs/src/main/resources/content/_configuring/content-directory-monitor.adoc

```diff
@@ -21,9 +22,15 @@ Configure the CDM from the ${admin-console}:
 . Select the *Configuration* tab.
 . Select *Catalog Content Directory Monitor*.
 
+.Content Directory Monitor Permissions
+[IMPORTANT]
+====
+If changing the monitored directory directory from the default, make sure to set permissions on the new directory to allow ${branding} to access it.
```

Comment:
`directory` is repeated

Reply:
❗️ We should actually mention in which section of the policy file it should be added and probably give an example

Reply:
Can you send me that info on the policy file? 

Deleted repeated word.

Reply:
@coyotesqrl She we be very specific about the code base this would be required in? Not sure we would want to allow all DDF access to that directory. It will mean that the code for directory monitoring should be modified to run with an access privileged block though.